src/main/java/com/gbtec/Cell.java

```diff
@@ -0,0 +1,18 @@
+package com.gbtec;
+
+public class Cell {
+
+    private boolean isAlive;
+
+    public Cell() {
+        isAlive = false;
+    }
+
+    public boolean getStatus() {
+        return isAlive;
+    }
+
+    public void setStatus(boolean status) {
+        isAlive = status;
+    }
```

Comment:
To improve the code readability in other classes, you can add methods like "void killCell()" and "void createCell()".